### DISCUSSION TITLE
Welcome to riscv-debug-spec Discussions!

### DISCUSSION THREAD

**Discussion by pdonahue-ventana** [riscv/riscv-debug-spec#1095] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HarinandanAM** [comment]
Hi everyone 👋

I’m currently working on RISC-V–based SoC development, mainly focusing on FPGA prototyping and microarchitecture-level exploration. My work involves integrating RISC-V cores, debug modules, memory subsystems, and studying how these designs transition from FPGA to ASIC.

Recently, I’ve been exploring the RISC-V Debug Specification in more depth — especially around JTAG, cJTAG, and possible lower pin-count alternatives for silicon implementations. I’m particularly interested in practical ASIC considerations such as stability, DFT impact, and long-term maintainability of debug interfaces.

I’m here to learn from the community, exchange ideas, and better understand real-world implementation experiences. Looking forward to engaging with you all!

**Comment by bruceable** [comment]
The RISC-V Debug Specification has a section on cJTAG and the recommended
MIPI 10-pin pinout.
